_posts/research/2023-03-24-hierarchical-ad-hoc-agents.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "HAHA: Hierarchical Ad Hoc Agents"
+description: Using hierarchical structures to improve human-agent interaction
```

Comment:
Make it less of a repetition of the title and more of a catchy / inspiring sentence